public/app/config/mapping.js

```diff
@@ -1,31 +1,64 @@
+  },
+  'https://utwente.graasp.eu/preproduction/refquest/build/quizTool.html': {
+    url:
+      'https://utwente.graasp.eu/preproduction/refquest/build/quizTool.one.html',
+  },
+  'https://utwente.graasp.eu/preproduction/tableTool/build/tableTool.html': {
+    url:
+      'https://utwente.graasp.eu/preproduction/tableTool/build/tableTool.one.html',
+  },
 
-  'https://utwente.graasp.eu/production/calculator/build/calculator.html':
-    'https://utwente.graasp.eu/production/calculator/build/calculator.one.html',
-  'https://utwente.graasp.eu/production/inputBox/build/inputBox.html':
-    'https://utwente.graasp.eu/production/inputBox/build/inputBox.one.html',
-  'https://utwente.graasp.eu/production/hypothesis/build/hypothesis.html':
-    'https://utwente.graasp.eu/production/hypothesis/build/hypothesis.one.html',
-  'https://utwente.graasp.eu/production/nameTheFrame/build/nameTheFrame.html':
-    'https://utwente.graasp.eu/production/nameTheFrame/build/nameTheFrame.one.html',
-  'https://utwente.graasp.eu/production/observationTool/build/observationTool.html':
-    'https://utwente.graasp.eu/production/observationTool/build/observationTool.one.html',
-  'https://utwente.graasp.eu/production/refquest/build/quizTool.html':
-    'https://utwente.graasp.eu/production/refquest/build/quizTool.one.html',
-  'https://utwente.graasp.eu/production/tableTool/build/tableTool.html':
-    'https://utwente.graasp.eu/production/tableTool/build/tableTool.one.html',
+  'https://utwente.graasp.eu/production/calculator/build/calculator.html': {
+    url:
+      'https://utwente.graasp.eu/production/calculator/build/calculator.one.html',
+  },
+  'https://utwente.graasp.eu/production/inputBox/build/inputBox.html': {
+    url:
+      'https://utwente.graasp.eu/production/inputBox/build/inputBox.one.html',
+  },
+  'https://utwente.graasp.eu/production/hypothesis/build/hypothesis.html': {
+    url:
+      'https://utwente.graasp.eu/production/hypothesis/build/hypothesis.one.html',
+  },
+  'https://utwente.graasp.eu/production/nameTheFrame/build/nameTheFrame.html': {
+    url:
+      'https://utwente.graasp.eu/production/nameTheFrame/build/nameTheFrame.one.html',
+  },
+  'https://utwente.graasp.eu/production/observationTool/build/observationTool.html': {
+    url:
+      'https://utwente.graasp.eu/production/observationTool/build/observationTool.one.html',
+  },
+  'https://utwente.graasp.eu/production/refquest/build/quizTool.html': {
+    url:
+      'https://utwente.graasp.eu/production/refquest/build/quizTool.one.html',
+  },
+  'https://utwente.graasp.eu/production/tableTool/build/tableTool.html': {
+    url:
+      'https://utwente.graasp.eu/production/tableTool/build/tableTool.one.html',
+  },
+  'https://apps.graasp.eu/5acb589d0d5d9464081c2d46/5d84ba5978f7f92b3d219b60/latest/index.html': {
+    main: 'index.html',
+    name: 'file-drop',
+    url: 'http://localhost/build/build.zip',
```

Comment:
Does the `localhost` mean we're prepackaging the file drop in the desktop viewer?